bazelrio/libraries/ni-libraries/chipobject/BUILD

```diff
@@ -1,7 +1,7 @@
 cc_library(
```

Comment:
Toss a `linkstatic=1` in your gen script. It will silence all of those warnings

Reply:
The gen script doesn't generate these files. It currently only modifies deps.bzl